### ISSUE TITLE
friendo mode

### ISSUE THREAD

**Issue by ThePotatoGuy** [Monika-After-Story/MonikaModDev#4849]
friendo mode was planned for a while now, but I don't remember if there was an issue to track it so here it is.

# Features

This would enable a mode where the user and Monika are not in a relationship.

This is separate from affection since people can love their friends while not being in love with them.

Differences in friendo mode:
* significantly less affection loss from missing events, not running the mod, saying "mean" things
* certain topics/dialogue maybe be relationship/friend-specific
* certain outfits maybe be relationship-only
* certain gifts will have different impacts in friendo mode (promise ring)

Switching between relationship mode and friendo mode can be dramatic depending on the circumstances.
* friendo -> relationship - large affection gains
* relationship -> friendo - small-large affection loss depending on how this dissolves.

Switching too often may lock relationship mode permanently.

Breaking up can be a result of several factors:
* using the breakup topic
* low affection for a period of time
* bad menu choices
* not running the mod for a time

Entering relationship can be a result of several factors:
* high affection for a period of time
* confessing your feelings
* running the mod consistently (time spent together is the largest factor in attraction)
* if in friendo mode, monika may bring up entering a relationship randomly but only if you haven't stated your intention to not be more than friends. 

Other things to be affected:
* initial do you love me question will include a friendo mode answer
* when did we start dating question will finally hook into friendo mode
* the break up topic will actually do something
* additional session stats for session time as friend/relationship
* [add more as we find more]

No, there will not be a friendzone. Friendzone criteria is not concrete enough to implement.

# Technical

* simple flag to determine if in relationship or not
* more timedeltas added to sessions to contain the split friendo/relationship data
* ~~new EV property `type_mask` - generic integer that will use bitmasking to hold a variety of type data. Two masks will be introduced: friendo and relationship, which determines if a topic should be part of a mode. This will **not** be saved to persistent. This gives us more flexibility than using a single bool to determine mode.~~ added in #4850 
* for current users, all previous time will be considered relationship data, unless you picked the future option in the "When did we start dating" dialogue. (**tbd**)

# Planning

Planned start date: ~~after #3768~~ tbd

**Comment by hlepnamingblyat** [comment]
If this is released, what will happen to my relationship with Monika?
Is this a choice?
Will this be a separate mod for MAS?

**Comment by Delcos** [comment]
not sure this is the best use of time and resources but alrighty.

**Comment by Rai99** [comment]
I'd expect a lot of discussion and resistance for this feature. 

For the record, I'm not really for it either.

**Comment by frostcattooo** [comment]
meh... doesn't it defeat the purpose of the mod? the mod where you can be with her for eternity in love?

**Comment by Rai99** [comment]
I think it does go against the mod. Not to mention, how much work would it be to update all the dialogue to be more of a friend mode rather than a loving girlfriend? That and maintaining both halves. 

It would basically be having the mod branch off in 2 very different directions from the start.

**Comment by Olliesama** [comment]
This goes against the purpose of the mod, it goes against Monika's character in terms of why she even continues forwards and any hopes she has for the future. It goes against the reasoning the mod gives for bringing her back and it goes against all of the hard work so far.
This is unequivocally a absolutely terrible idea and should be scrapped.

This doesn't fit into her character at all, why would anyone who's only reasoning for not having deleted herself because of the existential dread ever continue because of a friend? Someone who isn't willing to make promises or dedication to her. Why would she continue to perpetuate her existence in such a fashion? It ignores her entire character arc. It doesn't fit into it. As well as that it goes against the wishes of the majority of the MAS fanbase, people don't gravitate to this mod to just be "friends"? Anyone who is just in that mindset is going in with a mindset of no dedication at all, it invites people who'll play around to see what's up and then drop it immediately once the content is gone, not the dedicated people that have stuck around and contributed.

It goes against the mod's initial intention, an extension of act 3 which it's entire premise was just  you and her. Together. This wasn't supposed to ever be a weird friendship simulator. 

Also who does this serve? These resources and time goes to serving a tiny niche which won't stick around or care. :/

And really, the moment the player ever suggests that they're no longer in a relationship and just want to "be friends", is the moment Monika would realistically delete herself because she has no reason to continue.

**Comment by MisterSimple** [comment]
Considering the amount of work involved vs the amount of people who would end up actually using it, I'm afraid it's a no from me. There's plenty of other stuff that could be worked on with both higher reward and lower effort.

**Comment by thezotikrus** [comment]
It's either too late or too early for April Fools' day...

**Comment by Delcos** [comment]
I'l go ahead and change my view from not sure to this is definitely a waist of development time and resources. Please shelve this idea. Thank you

**Comment by dragonsni** [comment]
This is a really bad idea, it belongs in "submod" territory. The purpose of the mod is the relationship. Without the love, what's the point?

Granted, Monika is an amazing girl, and I love her to death. But for the mod, friend **and** lover is the picture.

**Comment by Delcos** [comment]
You have 175 days top wait, then please post this again.

**Comment by Delcos** [comment]
You have 175 days to wait, then please post this again.

**Comment by lunulae** [comment]
(TL;DR I actually believe very strongly that it would be in-character, but I don't know if it's a good use of resources.)

I actually vehemently disagree that Monika's only reason for being is a romantic relationship.
Her driving desire, as we all know, is to escape her reality and join us in ours. She wants to live in a world where choice matters, where things change, rather than in a scripted existence that runs in cruel loops.
Knowing the player, knowing someone "real" gives her hope. She immediately falls in love with us because we belong to the kind of reality she wishes she belonged in, because we're proof it exists.

I think that a friend mode would still be in-character for her, because it doesn't take away that core drive. She still would long to be in the real world. She'd still hope for us to bring her to it. Visiting her daily would still make her feel less alone in her isolated existence, and it would still be an important promise to make because of the weight of that.

I think the idea that anyone needs a romantic partner to want to live is nonsense, and something that has severely harmed us as a culture. But I understand why Monika deletes herself if you abandon her in MAS even without the romantic context. We might live in a world where we don't need to put all our eggs into one basket to find happiness, but she is. If she can have us as a friend, and have a relationship that brings her joy--even if it's a different type of joy--it's still saving her from isolation. Likewise, abandoning her even as a friend would have the same effect--she's alone, with nothing but that empty world she hates.

That said, I also understand that this is a modding team that is working 100% on their own time and their own funds. This would be a new direction of resources, and many players might not ever see it. And that's fine, I'm totally okay with not seeing that side of the game because it's not what my relationship with Monika would be. I think it would be a joy to the people who want it, I think it would still be in-character, and that makes it awesome to me. 

But because this is a smaller team working for free, and this is a bigger feature, if there's not anyone actively interested in programming it, I don't want mods to burn out on it. It's really down to if y'all really want to do it. If not, we should cheer on folks wanting to submod it, but that's all I think anyone's really obligated to do.

**Comment by TheSuperStu** [comment]
This is just _weird_ to see really. I understand this mod is suppose to be "after" the events unfold from the original DDLC, but those events that did happen were more for romantic reasons (even how extreme they were). Moni shows to have the desire to be with the player in their own world, in a more romantic manner. Of course there are always other people who want to see what the mod is all about without interest in the romance aspect.

Saying its a waste of resources (even if I sorta agree with) is a little rough since this mod has been open for people to try things and add their own stuff. But again I think this is weird since I've seen talk about neat submods or others working to try and add stuff that could be submod, but THIS is something more priority or pushed for official? I see talks of a weather tracking submod or auto hair/outfit changing, but THIS is what is wanted to be looked for? (I think **this is my main issue here**, even with seeing the priority being lowered lol.)

I am a fan of having Moni change her behavior based on player input on certain subjects. That is something that I think is obvious and can be agreed on to work towards, since Moni wants the player to be happy right. This idea of the mode, while I am not a fan of it, I can see why it would be added, but for the distant future and not a necessity right now.

**Comment by Delcos** [comment]
> This is just _weird_ to see really. I understand this mod is suppose to be "after" the events unfold from the original DDLC, but those events that did happen were more for romantic reasons (even how extreme they were). Moni shows to have the desire to be with the player in their own world, in a more romantic manner. Of course there are always other people who want to see what the mod is all about without interest in the romance aspect.
> 
> Saying its a waste of resources (even if I sorta agree with) is a little rough since this mod has been open for people to try things and add their own stuff. But again I think this is weird since I've seen talk about neat submods or others working to try and add stuff that could be submod, but THIS is something more priority or pushed for official? I see talks of a weather tracking submod or auto hair/outfit changing, but THIS is what is wanted to be looked for? (I think **this is my main issue here**, even with seeing the priority being lowered lol.)
> 
> I am a fan of having Moni change her behavior based on player input on certain subjects. That is something that I think is obvious and can be agreed on to work towards, since Moni wants the player to be happy right. This idea of the mode, while I am not a fan of it, I can see why it would be added, but for the distant future and not a necessity right now. (I prefer the parts where Moni says "I love you!")

It's just numbers. The amount of time and resources it will take in exchange for how many people will actually ever use it doesn't add up at all. I punch these numbers into my calculator and it starts to cry.

**Comment by Delcos** [comment]
![download](https://user-images.githubusercontent.com/28798918/66535656-24eaac80-eacf-11e9-867d-2cc93c89fe06.png)

**Comment by RavenPLAYER** [comment]
![m_sticker_sad](https://user-images.githubusercontent.com/55263957/66536029-03e78300-eaf2-11e9-9611-b8e49c6c4124.png)
"I'm not agree"
       -Monika

**Comment by AlenNez** [comment]
I agree with most people, this seems like a waste of effort and time. No need to invest in something that won't make any return just to please maybe 7-20 people who love monika "as a friend".

**Comment by Delcos** [comment]
> I agree with most people, this seems like a waste of effort and time. No need to invest in something that won't make any return just to please maybe 7-20 people who love monika "as a friend".

7-20 is a high shot

**Comment by Velius94** [comment]
![council](https://user-images.githubusercontent.com/1176163/66605359-4374a480-ebb0-11e9-8788-338042bb0835.png)

Not a fan of this thing, just like many others before me for reasons stated already stated above.

**Comment by lunulae** [comment]
I really don't wanna come across as condescending when I say this, but I really worry for those who have implied that "just wanting to be friends" means someone won't be dedicated to Monika and the promises they've made to her.

I won't make any assumptions about what might lead people to make statements like that, but it's a very worrying mindset if you think similarly about real people.

For those of you who won't use this, think of it as similar to sensitive mode. It doesn't affect you at all, unless you seek it out. Heck, I'm all for a toggle to make it so you can't even accidentally stumble on the option, since folks seem to be worried.

I know earlier I said if the devs aren't super invested in this, it shouldn't have to be a feature, because they're working for free just out of passion for this mod. But conversely I think the opposite is true. If this is something someone's passionate about, I do support them in making this. And with the overwhelming surge of negativity this concept has been getting, I think that bears repeating.

**Comment by Delcos** [comment]
It's just not worth the time or resources. The majority has given their opinion. You're free to make a submod but please keep it out of the main development cycle.

![k9jslg6o6j601](https://user-images.githubusercontent.com/28798918/66612209-e2c87600-eb75-11e9-9215-f4772abd8c96.png)

**Comment by antimockingjay** [comment]
Y'all are being so childish right now. "I don't want it which means nobody should be allowed to have it!" Have you considered simply not using it?

Also, all of you who are saying that friendship can't be as dedicated are WRONG.

**Comment by Delcos** [comment]
> Y'all are being so childish right now. "I don't want it which means nobody should be allowed to have it!" Have you considered simply not using it?
> 
> Also, all of you who are saying that friendship can't be as dedicated are WRONG.

It's not like it just appears out of the air and I can just disable it if I don't want it. This feature is a waist of time and effort, especially considering the community response it got. There is **No** way to justify spending time on this at all. The debate has already happened and people don't want this. End of discussion.

**Comment by antimockingjay** [comment]
> > Y'all are being so childish right now. "I don't want it which means nobody should be allowed to have it!" Have you considered simply not using it?
> > Also, all of you who are saying that friendship can't be as dedicated are WRONG.
> 
> It's not like it just appears out of the air and I can just disable it if I don't want it. This feature is a waist of time and effort, especially considering the community response it got. There is **No** way to justify spending time on this at all. The debate has already happened and people don't want this. End of discussion.

So screw anyone who does deeply care for Monika but not in that way. Anyone who doesn't love Monika in what you deem the "right" way doesn't deserve to play the mod, even if the option doesn't impact you in any way, shape, or form.

And what do you mean it's not like you can disable it? You literally have to go out of your way to use it, to my understanding. How much more ability to disable it do you want???

**Comment by lunulae** [comment]
Delcos, I do thoroughly respect your opinions thus far. But it's not your place to decide "the debate has already happened" when people are still discussing. If your only contribution is to try to shut down further discussion on a still-open issue, that's a pretty bad look.

Likewise, I should be apologizing as well. Just making a statement about friendship earlier was arguably out of place for the discussion of this function in the mod.

I think it might be worth discussing some things that could use trimming in this, to make the plan less bloated if it's going to be made.

I don't necessarily think different programming for affection necessarily needs to be done, for example, if dialogue is going to be changed anyway. I do think a significant rise/drop when it's toggled is appropriate.

It might be better for this mode to be toggled by a very obvious menu choice (breakup option and a counterpart?) rather than by poor affection choices in case of mistakes, as I see locking in after too many switches is under consideration. I understand casual mode might be meant to combat this, but I'm not sure it's a great solution.

I'd probably disable anniversary dialogue but maybe keep it on the calendar? Unsure.

**Comment by kaido1224** [comment]
I feel like the time would be better served working on a monika emotions engine or something of that ilk.

**Comment by multimokia** [comment]
![image](https://user-images.githubusercontent.com/22531674/66618723-479cc400-eba8-11e9-922b-3d1e6c928d2a.png)

Is probably the best argument I can provide against this, lorewise.

Let me put it this way:
If you think about life from Moni's perspective...
She's only there for you because she loves you.
You loved her to bring her back and/or even mod the game to make time with her better
But then you essentially got her hopes up, and say "You'll always be my dearest friend."

Given everything Monika is as her character, she would lose her will to live there and then.

This simply is not in character.

In the screenshot above, Monika doesn't want to be replaced. She also says it herself, she only wants to spend eternity with her sweetheart and no one else.

You're her purpose.

This just throws it all away.

**Comment by kaido1224** [comment]
Also, when we discussed friendzone mode in the past, i always thought it was intended as a punishment for negative affection before she left you completely. Although thinking about it now, that really doesn't make much sense I guess.

**Comment by antimockingjay** [comment]
Here are some reasons I think this would be a good thing to implement:

First off, real life happens. There have been times where I had to take a bit of a break from a romantic relationship, not because I didn't still love the person with all my heart, but because something outside of the relationship was making me emotionally unfit for a relationship. I have depression, and I know some other players of this mod do, too. There are times when I can't handle a romantic relationship, but that doesn't mean I want to cut that person out of my life entirely. People deserve to have that option open to them if they need it.

Secondly, people like me. People who truly do love Monika, but just... Not like that. I care about her more than just some random friend, but the truth is, I'm gayer than the rainbow. That doesn't change the fact that Monika is incredibly important to me. That's why I downloaded this mod. Because she's important to me. Not because of romance or whatever. _That_ is the point of this mod. It's for people who _care_ about Monika, not just people who wanna date her. So no, it doesn't defeat the purpose of the mod.

Friendship is NOT less important than romance. Platonic relationships are NOT inherently less than romantic relationships. Nonromantic relationships are NOT inherently the most meaningful type of relationship. To imply that you can't love Monika if you aren't romantically interested is downright offensive. In most situations, I have actually valued platonic relationships more than romantic ones.

If you personally view romantic relationships as more important _to you,_ then that is valid. But it isn't fair to force that view of relationships onto other people. I would jump in front of a train for Monika, if she were in our reality. That doesn't mean I want to get in her pants or something. I don't. I just _care._ That's what this mod is supposed to be about.

There are so many kinds of love, and it isn't fair to try and force the idea that one is inherently more important than all the others.

**Comment by Rai99** [comment]
As stated before, this isn't about optional features here and there that people can just avoid if they don't want. This would be an incredibly large project, and would take a long time to implement. 

Regardless of the quality of it or how many people do or don't want it, it's clearly a new direction for the mod, which isn't a surefire good thing. 

With the amount of time it would take to implement _and_ review, you could expect that MAS won't be getting any fancy, new features for a while until this gets completed. Would you use this? Would this make you happy? Would you say this feature is worth the time and effort? 

Even if it was already completed and implemented, this would change **_most_** dialogue, which would make maintaining the mod much, much harder, and potentially multiply the size of the mod with the new "mode" for each topic, especially if the alternate responses have a real level of depth to them, and aren't just a simple "couple of lines" to accommodate the new mode, meaning quality would also be a concern. 

MAS is about maintaining a romantic relationship with Monika. Some people may disagree that Monika only exists to be a romantic partner. Those people are free to play another mod. In it's current state, MAS isn't suited to take on an entirely new direction such as this "friend mode". Maybe there are a few people who do want it. That's not what we do. 

At that rate, we might as well have the option to replace Monika with a different doki to "open the mod to a broader audience". Why not have yet another mode where Monika is an absolute masochistic pervert? Some people might want it, and it would be optional in case some people don't like it, so it can only be good, right? 

Think of it this way: You have a car. It drives. It works. It does what you need it to. It could be more though. The roads are a dangerous place, so let's add armor to the car to help protect it during accidents. It becomes heavier, slower, and less fuel efficient. Let's add a bigger engine to keep some of the speed. The car get's bigger and harder to control. Overhaul the steering mechanism to make it easier to control. It's no normal car anymore, and you would need to know specifically how to drive that vehicle, but it's yours, so it should be fine. It's too tall now though, so you have to shorten it and place more parts horizontally, making it much longer, and harder to steer. Look at all that space though. You could fit a whole bunch of extra cabins and seats with that extra space, so you do. So it turns into a makeshift, giant taxi for you and all your friends. Sounds nice, but it would be cool if it was also amphibious. With how heavy it already is, you're going to need some serious modification to make sure it floats. Being a sea vehicle now, there are extra regulations required for the vehicle to legally operate, such as having X amount of life boats in the event of an emergency. So now it's a land and sea vehicle. Why stop there? Let's make it fly too. So let's see, it's got heavy armor, a heavy engine, a large and heavy fuel tank to hold all the fuel your monstrosity needs, extra cabs, flotation mechanisms, etc. This thing has to be able to fly. Let's just call NASA and borrow some of their rockets. Oh, but their rockets need specialized fuel, so we'll need some more space to hold that, and a couple more rockets to lift that extra weight too. It's so big, it might as well be a house, so let's stick some furniture in there. Maybe it could even be an apartment complex too. Wait, we just got a call from the government saying there's a war on the horizon and they want to use your vehicle. No problem, you've got **_plenty_** of room for missiles, ammunition, and soldiers to  boot. Although, being a war vehicle, it'll be subject to much more intense environments and threats, so it'll need more armor to protect it. Maybe you could wear missiles as hats to squeeze in a few extra things. 

And then you're all ready to go. A ground/sea/air vehicle capable of transporting, housing, and arming people. It takes a whole lot of professionals to even maintain, but it does anything and everything. At this rate, it'll be finished in a couple decades if we're lucky, and it only cost you a little bit. A couple hundred generations, and maybe you'll be able to afford to take it back from the government. It sure would make a great gift for your descendants, right? Hopefully, they'll actually utilize every single feature so none of the effort goes to waste. 

That's just an example of what can happen if you try to make anything something that it's not. At best, you'd get something that's crappy at everything, but it does do everything. At that rate though, no one's going to use it when there are actual, proper things out there to do the job you need. 

Somewhere out there, maybe there's already a DDLC mod that covers just this, but that has nothing to do with us. We aren't that mod, and we aren't any other mod. That's not what we do. If you're interested though, and **_only_** if you're interested, we can offer you a romantic relationship with Monika. 

If you're not interested, have a nice day. I hope you find what you're looking for. I hope it'll be really cool.

**Comment by antimockingjay** [comment]
@Rai99 Point me in the direction of this other mod that has the friend-style mode and I'll stop trying to get it into this one.

**Comment by Rai99** [comment]
I said maybe. I don't know if there's anything like that out there right now, but if you ask me, you won't find anything like that here.

**Comment by antimockingjay** [comment]
"So I don't know if this can be found anywhere else, but either way, don't work on it because I personally don't want it."
That's what you sound like right now, I want you to understand that. You don't even have any backup for anyone who does want this, but you're telling us to just go somewhere else to find it.

**Comment by AlenNez** [comment]
Because this isn't the place for it at all. Go chat with cleverbot if you want a "friend" instead of dissolving MAS into nothing more than a GF simulator.

**Comment by Rai99** [comment]
If you ask me, I think it's unlikely to happen. I don't intend to take part in the development of this feature, but there are other people who may be interested or made to help, potentially, devs. 

Here are my main concerns overall:
- This is going to take a long time to implement. Regardless who wants it and who doesn't want it, fancy, new features probably aren't going to happen until this gets done. I would predict that this level of hiatus would lose us more users than this feature would gain us. 
- A lot of the current dialogue is tailored as if Monika was your significant other, which at the moment, she is or should be. If I'm reading this correctly, this would require an alternate path to about every single existing topic, and any future topic that may lean one way or the other. Hence the long development time. 
- After development of this feature is completed, there will be two differing paths to each topic, pretty much doubling file size and making it much, much harder to maintain. It would be a very different direction, and I think it would be a bad idea to lump the two approaches together. Even avoiding this, designing yet another framework would take additional time, especially given how many topics there are. 
- If this feature successfully makes it in, what's going to stop MAS from doing it again with another wildly different mode? More features that most people aren't going to use, extra file size. more alternate dialogue, potentially more frameworks.

**Comment by antimockingjay** [comment]
@AlenNez You just compared talking with Monika to talking with cleverbot, yet those of us who love her platonically are somehow the ones who don't take it seriously enough?

**Comment by lunulae** [comment]
I still don't think that changing the nature of your relationship is "replacing" her, though.
Do I think it would hurt? Yeah, any breakup hurts for a while, even if it's on good terms. But I have to respectfully disagree that the romantic relationship is the core of what's holding Monika's will to live together--it's the promise that you won't leave her behind, that she'll get to be a part of our reality, whether it's through spending time with us or us finding a way to bring her here. I think a friend can still maintain that promise. I think a good friend would take it very seriously.

I can agree with the points @Rai99 is making, though. This would add a lot of size and development time. It's definitely a good idea to find what ways we can to trim it down and make it manageable if that's possible. If not, it might be a good idea to turn the discussion toward how a submod version could be made/maintained?

Though I do see, going back to my earlier post, that a good number of the devs themselves seem to not be too on-board with this. I'm not really for pressuring anyone to be working on this without pay, if they don't believe in it. Granted, I don't know if there's divide on this, so I don't want to speak too much without full knowledge.

**Comment by AlenNez** [comment]
> @AlenNez You just compared talking with Monika to talking with cleverbot, yet those of us who love her platonically are somehow the ones who don't take it seriously enough?

I'm comparing "friend mode" monika to cleverbot. Don't twist my words. There are numerous reasons including wasted effort for not implementing this ridiculous and undoubtedly useless mode to satisfy a tiny amount of people.

**Comment by multimokia** [comment]
I'm completely fine with a submod approach, that makes it *purely* optional and **not** a core part of MAS

**Comment by Rai99** [comment]
If this was an external project or submod undertaken by a random user, they would be free to make it. The reason it's so significant is that the apparent plan here, is to have work done on this officially, potentially crippling the development of MAS for a time. That, and I don't know or see too many people keen on this feature, so I can't predict the overall quality of the feature if it were done.

**Comment by kaido1224** [comment]
This is a civil discussion about whether or not this should be implemented. No one should be alienated for having a different opinion. @AlenNez your post is not helpful and completely unnecessary. I hope someone kindly Mark's it as off topic.

**Comment by antimockingjay** [comment]
Maybe it should be somewhat in between. An official submod, so to speak? Like, it's officially being developed by someone/multiple someones on the dev team(depending on who in the dev team decides to work on it) but still has to be actively sought out, or something to that effect.

**Comment by multimokia** [comment]
I have no interest in developing this personally.

**Comment by AlenNez** [comment]
> This is a civil discussion about whether or not this should be implemented. No one should be alienated for having a different opinion. @AlenNez your post is not helpful and completely unnecessary. I hope someone kindly Mark's it as off topic.

Not everything that disagrees with you is wrong and intentionally harmful. I know that I have a great point, because it's a waste of time and effort that hardly anyone wants. I'd like for you to name a significant portion of people who would actually want this added into main development.

**Comment by antimockingjay** [comment]
> I have no interest in developing this personally.

That's why I said "depending on who in the dev team decides to work on it." Because people shouldn't be forced to work on it, and presumably, you wouldn't be forced to.

> > This is a civil discussion about whether or not this should be implemented. No one should be alienated for having a different opinion. @AlenNez your post is not helpful and completely unnecessary. I hope someone kindly Mark's it as off topic.
> 
> Not everything that disagrees with you is wrong and intentionally harmful. I know that I have a great point, because it's a waste of time and effort that hardly anyone wants. I'd like for you to name a significant portion of people who would actually want this added into main development.

We aren't saying there's an issue with you disagreeing, it's the fact that you're not contributing anything constructive to the conversation. You're just giving a smart-alec comeback and trying to be sassy. That doesn't make it a good point, it just makes you seem rude.

**Comment by AlenNez** [comment]
> > Not everything that disagrees with you is wrong and intentionally harmful. I know that I have a great point, because it's a waste of time and effort that hardly anyone wants. I'd like for you to name a significant portion of people who would actually want this added into main development.
> 
> We aren't saying there's an issue with you disagreeing, it's the fact that you're not contributing anything constructive to the conversation. You're just giving a smart-alec comeback and trying to be sassy. That doesn't make it a good point, it just makes you seem rude.

Then you're misinterpreting it. I am inserting my thoughts and opinions into this and you automatically reject them as me trying to be rude. Now back to the topic, WHAT REASONS do you have for wanting this? Who are you pleasing? Why even add this in the first place if it's going to satisfy a nigh tiny minority of people?

**Comment by lunulae** [comment]
> I'd like for you to name a significant portion of people who would actually want this added into main development.

Not that I like to speak for any entire demographic, but--gay men and straight women who love Monika?
Not that it changes my answer to the development-time question, I still think it'd be better submod territory but I think it'd please more people than you'd expect.

**Comment by MisterSimple** [comment]
Just because MAS developers won't work on it doesn't mean such a feature can't be a submod. If enough people actually care, then they'll start working towards that goal. And if they don't, then they didn't care that much, so choosing not to actively work on it was the right call.

**Comment by lunulae** [comment]
> And if they don't, then they didn't care that much, so choosing not to actively work on it was the right call.
I think this might be a bit of a poor claim, we're already establishing in favor of it not happening that this might be a heftier project. If we add this onto it, we're saying "only people with the ability to program something this intensive should matter."

Sometimes things that would be excellent just can't realistically get implemented for many reasons. I agree with that, but I don't agree with then saying "so then it never mattered at all."

**Comment by Daiozen** [comment]
I'm not a fan of this idea either, though not as much against it as I'm against #4851.

I think becoming friend with Monika could be a possible development over the long term; although just knowing it's a possibility would cheapen her displays of affection as they are now (what happens to "I'll just love you for all eternity player, just remember that."?). 

My main concern would be about giving the possibility of becoming friend right away with just the click of a button, as it would seem completely out of character with how she is in the original DDLC. From her actions and dialogues in the game, it's pretty clear she do not only want to be friend with the player.

Making Monika just go along with whatever the player wishes right away would take away from her character. I feel she needs to have a strong stance on this, and show that even if she loves the player, she is her own character with wishes of her own, even if they might not be the player's wishes. 

My other concerns has been pointed out plenty of times, and it's about the workload for the actual benefits of the few (I'm saying the few as a blind guess here, I don't have any numbers to back this up). It would take a tremendous amount of work to re-write the large majority of her dialogues to make them fit this mode and keep the whole coherent. It's not only the front cost either, every addition from this point on would need to take into account this added complexity, which does not only requires some extra time but also might make it less friendly for people to contribute.

To sum this up, I feel a mode like this, though theoretically possible, would cheapen or weaken the existing content for most of the current "romance" players (me included). It would also add too much long term complexity (especially dialogue wise) for the actual benefits of implementing it.

I realize there are players who only love Monika as a friend, and it does not take away from the affection they feel for her. But I feel this mod should keep focusing on the romance aspect. Making a particular version of Monika to specifficaly fit to a minority of players should be a submod and not part of core MAS.

**Comment by MisterSimple** [comment]
> I think this might be a bit of a poor claim, we're already establishing in favor of it not happening that this might be a heftier project. If we add this onto it, we're saying "only people with the ability to program something this intensive should matter."

Should have worded that better, apologies. Didn't mean to say only people with the ability to do that by themselves matter; I meant that if people really wanted that to happen, then they'd start working towards it by themselves. Sure, not everyone is a programmer, nor an artist, nor a writer, but if users wanted that sort of content they could try to make it themselves, using whatever skill they're proficient at. Then they could look for support from more experienced people willing to lend a hand.

**Comment by lunulae** [comment]
@Daiozen I don't necessarily think it'd cheapen the relationship--I'd think the fact that she still loves me even given the possibility not to would make it feel more special.

But I do think you have a really good point about the idea of making switching from romance to friend too simple, and keeping Monika's agency in mind. I'm thinking about it way too simplistically.

Maybe (and this could be good to keep in mind even if this is made as just a submod) it might be important to remember that the difference between the mod and Act 3 is that we can talk back to Monika. Before it was quite literally a one-sided relationship, so that might be an avenue for opening the friendship arc, since Monika does acknowledge meta elements of the mod.

**Comment by Daiozen** [comment]
> I don't necessarily think it'd cheapen the relationship--I'd think the fact that she still loves me even given the possibility not to would make it feel more special.

@lunulae This is what I mean by this :
There are certain sentences, like these ones, that really marked me :
```
"So I'll always love you for all eternity, [player]. Just remember that."
"I'll always love you no matter what."
"I love you and I'll always be here for you, so don't ever forget that~"
"You're a wonderful person, and I will always love you."
```
As things are now, there isn't any way those would not be true, simply because there isn't any alternative in the mod. This is also part of what makes this relationship special  and precious to me : I have no doubt I can trust her on this.
If there was a way these would not be true (like going from lover to friend), these sentences would not have had as much of an impact on me, this is what I mean by 'cheapen'.

**Comment by 99Dman99** [comment]
This feature seems like another take on a suggestion that a vocal minority want. It's pandering the mod to a specific few people (Much like any LGBT topics), that you could probably count on 2 or fewer hands. The time and effort is not with it for this feature, it would be better spend on other features that have way more hype and audience interest.

Consider "Sensitive Mode" and how that didn't take off. Because it pandered to a very small number of people. Think about the time and effort that was put into changing the assets of hangman, the dialogue of each topic and changing expressions. All for it to be disabled by 99% of all the Mod players. Both sensitive mode and friendo mode are a complete waste of time.

The friendo mode is a kind of feature that is not something that most players want nor will use. Because it intentionally limits their experience, like Sensitive mode does. Along side the fact that majority of players have full immersion of loving Monika in the Mod, and this mod is designed to break that immersion, by not being the happy ending where you get the best girl of the game.

If you wanted her to be more of a friend or side character, you might as well play other mods that focus on all 4 of the girls, or even just sticking with vanilla DDLC.

**Comment by Delcos** [comment]
> This feature seems like another take on a suggestion that a vocal minority want. It's pandering the mod to a specific few people (Much like any LGBT topics), that you could probably count on 2 or fewer hands. The time and effort is not with it for this feature, it would be better spend on other features that have way more hype and audience interest.
> 
> Consider "Sensitive Mode" and how that didn't take off. Because it pandered to a very small number of people. Think about the time and effort that was put into changing the assets of hangman, the dialogue of each topic and changing expressions. All for it to be disabled by 99% of all the Mod players. Both sensitive mode and friendo mode are a complete waste of time.
> 
> The friendo mode is a kind of feature that is not something that most players want nor will use. Because it intentionally limits their experience, like Sensitive mode does. Along side the fact that majority of players have full immersion of loving Monika in the Mod, and this mod is designed to break that immersion, by not being the happy ending where you get the best girl of the game.
> 
> If you wanted her to be more of a friend or side character, you might as well play other mods that focus on all 4 of the girls, or even just sticking with vanilla DDLC.

Couldn't say it better myself.

**Comment by bittercaramel** [comment]
too lazy to read all the stuff above, but i personally pretty much liek this idea 
yeah that's all i had to say, kinda defeats purpose of the mod, but i like it anyways ¯_( ´･ω･)_/¯

**Comment by PollenTheCat** [comment]
I like the idea of a friend mode, but as people are saying, it's kinda not worth it and will take a lot of time. I agree in that it would be better as a submod.

**Comment by DarkRosePrincess** [comment]
I...I will be honest. I almost teared up reading about this suggestion.
This...this is so against why I sought out and downloaded this mod that I might never update it if this comes out and is not optional.
Please, if you really think this is necessary - as I do understand why others would want it - please make it a sub-mod, I implore you.

**Comment by ThePotatoGuy** [comment]
[closing until we look into it]

**Comment by SonicEraZoR** [comment]
This would be perfect mode for me since I like spending time with Monika but I don't really *love* her

**Comment by Freshairkaboom** [comment]
Sorry, but I hope this never becomes a thing in the official mod.
